Backend/app/rider-platform/rider-app/Main/src/API/UI/Ride.hs

```diff
@@ -32,14 +40,68 @@ import Tools.Auth
 
 type API =
   "ride"
-    :> Capture "rideId" (Id SRide.Ride)
-    :> "driver"
-    :> "location"
-    :> TokenAuth
-    :> Post '[JSON] DRide.GetDriverLocRes
+    :> ( Capture "rideId" (Id SRide.Ride)
```

Comment:
We have changed the SOS/share ride flow at ride side.
Now, emergency number will be only at person level that you have already added. At ride state, we don't need to share ride to specific contact, it will be an open api.

So, you need to do following changes
1. Remove emergency contacts related api from ride 
2. Only have 2 APIs which should be open without any authentication a) get driver location with ride status b) get full ride details same as GetRideStatusResp

Reply:
We don't need any emergency level flow at ride side.